Update dependencies

gaetan-h opened the pull request.

cognite-cicd commented:
#### Checkmarx scan completed with the following findings
|Lines|Severity|Category|File|Link|
---|---|---|---|---

gaetan-h commented:
- Cloned repository
- Updated pyproject.toml file
- Deleted poetry.lock file locally
- Ran poetry update
- Commited pyproject.toml and poetry.lock files

gaetan-h commented:
Updated poetry version as well to make the build pass
